### DISCUSSION TITLE
Clarification about loss weights

### DISCUSSION THREAD

**Discussion by isfons** [lululxvi/deepxde#1861] [Q&A]
Hi everyone,

I am trying to solve an inverse problem using DeepXDE and I have doubts about how to specify the loss_weights argument in the compile method. It is not clear to me in which order should I provide the list so that the proper weight multiplies the right loss term.

Could anyone help on that, please?

**Comment by lululxvi** [comment]
The same order as the losses defined in the data class such as `dde.data.PDE`.

**Reply by isfons** [reply]
Ok, thanks for clarifying it!
